### PR TITLE
Feature/#42 exclamations

### LINKED ISSUE #42: Show in the view animal exclamations

### PR DESCRIPTION
**Pull request by DominikNowak**
closes #42 

merge after #41

### CODE REVIEW

**Review by Enessetere — APPROVED** [review]
Looks good to me ;) Well done :)

File: src/Animals/Wolf.ts [hunk 1]
```diff
@@ -10,6 +11,8 @@ export class Wolf extends Animal implements AttackHerdInterface {
 
   // TODO: implement when herd is ready
```
Comment (synowa) [inline src/Animals/Wolf.ts:12]: Maybe this comment is no longer needed ;)

File: src/Animals/Wolf.ts [hunk 2]
```diff
@@ -10,6 +11,8 @@ export class Wolf extends Animal implements AttackHerdInterface {
...
   attackHerd(): string {
-    return `${this.name}: Auuuuuu! I ate all animals in the herd except horses and small dog. Grrrrr!`;
+    const exclamation = `${this.name}: Auuuuuu! I ate all animals in the herd except horses and small dog. Grrrrr!`;
```
Comment (synowa) [inline src/Animals/Wolf.ts:14]: I like how specific the wolf is 😆

**Review by synowa — APPROVED** [review]
I think everything is fine.

**Pull request merged**